src/options.cpp

```diff
@@ -1491,13 +1491,13 @@ void options_manager::init()
 
     mOptionsSort["world_default"]++;
 
-    add( "STATIC_NPC", "world_default", translate_marker( "Static NPC" ),
-        translate_marker( "If true, the game will spawn a static NPC at the start of the game, requires world reset." ),
+    add( "STATIC_NPC", "world_default", translate_marker( "Static NPCs" ),
+        translate_marker( "If true, static NPCs will spawn at pre-defined locations. Requires world reset." ),
```

Comment:
Sorry, missed this last time -- the comments into it.

Doesn't require world reset; player has to travel to a refugee center that has not been generated yet, and static NPCs will be present. Can easily be tested by making new world with Static NPCs off, make survivor, turn Static NPCs on, travel to nearest refugee center and should have NPCs (depending on distance of refugee center if it was loaded into reality bubble or not, I believe).

Not sure what to change to, perhaps `locations.  Does not affect explored areas.` Or maybe just keep it with world reset to avoid plethora of issues popping up "static NPC don't work" etc.

Reply:
> Or maybe just keep it with world reset to avoid plethora of issues popping up "static NPC don't work" etc

Going with this. Good notes, however.